README.md

```diff
@@ -1 +1,92 @@
-# angular-resource-readme
\ No newline at end of file
+# $resource
+
+## Overview
+
+We've had a look at the awesome `$http` service that Angular provides, but there is also another awesome service under the name of `$resource`. Let's take a look at that!
+
+## Objectives
+
+- Describe $resource
+- Use $resource methods
+- Fetch data from an API
+- Update data from an API
+
+## What is $resource?
+
+$resource is a factory that creates a resource object for us to communicate with APIs. We pass through a URL and it gives us an object back that looks like this:
+
+```js
+var User = $resource('/user/:userId');
+
+/**
+ * User = { get: function (),
```

Comment:
I like that you show what gets created for us here - this gives the user a peek at what `$resource` is doing for us.